### ISSUE TITLE
It's not possible to edit image details with a blank "File" form field

### ISSUE THREAD

**Issue by m1kola** [torchbox/wagtail#2897]
Browser validation doesn't allow to save image details if a "File" form field is blank.

I can reproduce it with Django 1.10 and Wagtail 1.6rc1. Everything works as expected with Django 1.9.9.

<img width="488" alt="screen shot 2016-08-08 at 14 36 14" src="https://cloud.githubusercontent.com/assets/509198/17478569/83e2b024-5d75-11e6-981f-fba5671ab9a9.png">

**Comment by m1kola** [comment]
Django 1.10 renders field like this:
```html
<input id="id_file" name="file" type="file" />
```

Django 1.9.9 like this:
```html
<input id="id_file" name="file" type="file" required />
```

**Comment by m1kola** [comment]
[Here](https://docs.djangoproject.com/en/1.10/ref/forms/api/#django.forms.Form.use_required_attribute) is the reason.

**It seems to that a similar problem may appear in other forms.**

**Comment by gasman** [comment]
As mentioned at https://github.com/torchbox/wagtail/pull/2899#issuecomment-238265450, I think this is a logical error at Django's end. Have now reported this in https://code.djangoproject.com/ticket/27037

**Comment by mevinbabuc** [comment]
Django has fixed the bug and made a [1.10.1 release](https://docs.djangoproject.com/en/1.10/releases/1.10.1/).

**Comment by m1kola** [comment]
@gasman it seems to me that we need to exclude `Django==1.10` from `install_requires`. What do you think of it?

**Comment by gasman** [comment]
Django 1.10.1 isn't released yet - nothing about it on https://pypi.python.org/pypi/Django or https://www.djangoproject.com/weblog/. Wagtail 1.6 final is due for release today, so I think we need to apply a workaround. I'd like to go with the approach in https://github.com/torchbox/wagtail/pull/2899#issuecomment-238533649 (with a custom tag to handle the 'if django==1.10' logic).

**Comment by m1kola** [comment]
Ah, Sorry. Documentation says "Under development", but fix is scheduled for 1.10.1. I tested it with their`stable/1.10.x` branch and it works.
